docs/content/how-to/common/exposing-services-from-hcp.md

```diff
@@ -0,0 +1,89 @@
+# Exposing the Hosted Control Plane Services
+
```

Comment:
can we start by introducing the different endpoints we expose and their motivations?